### DISCUSSION TITLE
Is all in one app being a good idea?

### DISCUSSION THREAD

**Discussion by chocolate-byte** [lencx/ChatGPT#733] [General]
For example, people won’t need to download pages of apps on their phone. They simply install one All-in one app based on charger.
Everyone runs this app with Siri or some stuff like that. 
I mean, they could run apps on cloud computer with almost doing anything.
Like, hi Siri, order a pizza from Sams street pizza hut. Then it shows the distance of your delivery on your iPhone’s Lock Screen.
Hi Siri, add these pages to my safari and divide them into my favorite. Then everything is done!
Hi Siri, get me a ticket, show me ticket to Hogwarts on Sunday. Then you got a ticket we code through your email, sms.

Basically runs everything online and save your phone for something more, forget all those stupid settings and not enough space , login stuff

the slogan of this app might be cool.
Don’t wanna install tons of apps and get bored with all the login, authentication code, tons of favorite on each apps? 
Worrying about having a date and appointment meanwhile, we could arrange you the best restaurant , find the best way and get a Uber.
Some other stuff like that…
Don’t worry, Chatgpt takes care of that, 
you just need to call them in one app, the godfather-Siri, XD
